<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelCalibrationWindow</class>
 <widget class="QWidget" name="ChannelCalibrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChannelCalibrationWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="currentChannelLabel">
       <property name="text">
        <string>Channel:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="currentChannelLabelDynamic">
       <property name="text">
        <string>&lt;channelNum&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="dataPointsLabel">
       <property name="text">
        <string>Number of data points to be used:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="dataPointsSpinBox">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="4">
      <widget class="QSpinBox" name="point6ReadChannelSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>7</number>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QDoubleSpinBox" name="point6yValuesDoubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>7</number>
       </property>
       <property name="minimum">
        <double>-99999.999999000006937</double>
       </property>
       <property name="maximum">
        <double>99999.999999000006937</double>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="point5xValuesDoubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>7</number>
       </property>
       <property name="minimum">
        <double>-99999.999999000006937</double>
       </property>
       <property name="maximum">
        <double>99999.999999000006937</double>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QDoubleSpinBox" name="point9yValuesDoubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>7</number>
       </property>
       <property name="minimum">
        <double>-99999.999999000006937</double>
       </property>
       <property name="maximum">
        <double>99999.999999000006937</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="point1Label">
       <property name="text">
        <string>Point 1:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QPushButton" name="point4ReadChannelButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Read V from DAQ</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="point8Label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Point 8:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QDoubleSpinBox" name="point10xValuesDoubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>7</number>
       </property>
       <property name="minimum">
        <double>-99999.999999000006937</double>
       </property>
       <property name="maximum">
        <double>99999.999999000006937</double>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QPushButton" name="point5ReadChannelButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Read V from DAQ</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QDoubleSpinBox" name="point7yValuesDoubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>7</number>
       </property>
       <property name="minimum">
        <double>-99999.999999000006937</double>
       </property>
       <property name="maximum">
        <double>99999.999999000006937</double>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="yValuesLabel">
       <property name="text">
        <string>y-values (&lt;unit&gt;)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QDoubleSpinBox" name="point4yValuesDoubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>7</number>
       </property>
       <property name="minimum">
        <double>-99999.999999000006937</double>
       </property>
       <property name="maximum">
        <double>99999.999999000006937</double>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QSpinBox" name="point5ReadChannelSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>7</number>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QDoubleSpinBox" name="point7xValuesDoubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>7</number>
       </property>
       <property name="minimum">
        <double>-99999.999999000006937</double>
       </property>
       <property name="maximum">
        <double>99999.999999000006937</double>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QDoubleSpinBox" name="point5yValuesDoubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>7</number>
       </property>
       <property name="minimum">
        <double>-99999.999999000006937</double>
       </property>
       <property name="maximum">
        <double>99999.999999000006937</double>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="point10Label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Point 10:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="point6Label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Point 6:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QSpinBox" name="point3ReadChannelSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>7</number>
       </property>
      </widget>
     </item>
     <item row="9" column="5">
      <widget class="QPushButton" name="point9ReadChannelButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Read V from DAQ</string>
       </property>
      </widget>
     </item>
     <item row="8" column="5">
      <widget class="QPushButton" name="point8ReadChannelButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Read V from DAQ</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="point3xValuesDoubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>7</number>
       </property>
       <property name="minimum">
        <double>-99999.999999000006937</double>
       </property>
       <property name="maximum">
        <double>99999.999999000006937</double>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="point1ReadChannelButton">
       <property name="text">
        <string>Read V from DAQ</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QDoubleSpinBox" name="point8yValuesDoubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>7</number>
       </property>
       <property name="minimum">
        <double>-99999.999999000006937</double>
       </property>
       <property name="maximum">
        <double>99999.999999000006937</double>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QPushButton" name="point3ReadChannelButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Read V from DAQ</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="xValuesLabel">
       <property name="text">
        <string>x-values (V)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="point5Label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Point 5:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="4">
      <widget class="QSpinBox" name="point10ReadChannelSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>7</number>
       </property>
      </widget>
     </item>
     <item row="0" column="4" colspan="2">
      <widget class="QLabel" name="readChanneLabel">
       <property name="text">
        <string>Channel to read from (optional)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="point1yValuesDoubleSpinBox">
       <property name="decimals">
        <number>7</number>
       </property>
       <property name="minimum">
        <double>-99999.999999000006937</double>
       </property>
       <property name="maximum">
        <double>99999.999999000006937</double>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDoubleSpinBox" name="point6xValuesDoubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>7</number>
       </property>
       <property name="minimum">
        <double>-99999.999999000006937</double>
       </property>
       <property name="maximum">
        <double>99999.999999000006937</double>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="point2yValuesDoubleSpinBox">
       <property name="decimals">
        <number>7</number>
       </property>
       <property name="minimum">
        <double>-99999.999999000006937</double>
       </property>
       <property name="maximum">
        <double>99999.999999000006937</double>
       </property>
      </widget>
     </item>
     <item row="6" column="5">
      <widget class="QPushButton" name="point6ReadChannelButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Read V from DAQ</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="point4xValuesDoubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>7</number>
       </property>
       <property name="minimum">
        <double>-99999.999999000006937</double>
       </property>
       <property name="maximum">
        <double>99999.999999000006937</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="point1xValuesDoubleSpinBox">
       <property name="decimals">
        <number>7</number>
       </property>
       <property name="minimum">
        <double>-99999.999999000006937</double>
       </property>
       <property name="maximum">
        <double>99999.999999000006937</double>
       </property>
      </widget>
     </item>
     <item row="7" column="5">
      <widget class="QPushButton" name="point7ReadChannelButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Read V from DAQ</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QDoubleSpinBox" name="point10yValuesDoubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>7</number>
       </property>
       <property name="minimum">
        <double>-99999.999999000006937</double>
       </property>
       <property name="maximum">
        <double>99999.999999000006937</double>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QSpinBox" name="point1ReadChannelSpinBox">
       <property name="maximum">
        <number>7</number>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QDoubleSpinBox" name="point8xValuesDoubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>7</number>
       </property>
       <property name="minimum">
        <double>-99999.999999000006937</double>
       </property>
       <property name="maximum">
        <double>99999.999999000006937</double>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QSpinBox" name="point4ReadChannelSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>7</number>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="point9Label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Point 9:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="4">
      <widget class="QSpinBox" name="point8ReadChannelSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>7</number>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="point2ReadChannelButton">
       <property name="text">
        <string>Read V from DAQ</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="point3Label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Point 3:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QDoubleSpinBox" name="point3yValuesDoubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>7</number>
       </property>
       <property name="minimum">
        <double>-99999.999999000006937</double>
       </property>
       <property name="maximum">
        <double>99999.999999000006937</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="point2xValuesDoubleSpinBox">
       <property name="decimals">
        <number>7</number>
       </property>
       <property name="minimum">
        <double>-99999.999999000006937</double>
       </property>
       <property name="maximum">
        <double>99999.999999000006937</double>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QSpinBox" name="point7ReadChannelSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>7</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="point4Label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Point 4:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="5">
      <widget class="QPushButton" name="point10ReadChannelButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Read V from DAQ</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="point7Label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Point 7:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="point2Label">
       <property name="text">
        <string>Point 2:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QDoubleSpinBox" name="point9xValuesDoubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>7</number>
       </property>
       <property name="minimum">
        <double>-99999.999999000006937</double>
       </property>
       <property name="maximum">
        <double>99999.999999000006937</double>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QSpinBox" name="point2ReadChannelSpinBox">
       <property name="maximum">
        <number>7</number>
       </property>
      </widget>
     </item>
     <item row="9" column="4">
      <widget class="QSpinBox" name="point9ReadChannelSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>7</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="slopeLabel">
           <property name="text">
            <string>Slope:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="slopeLabelDynamic">
           <property name="text">
            <string>Not calculated</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="yinterceptLabel">
           <property name="text">
            <string>y-intercept:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="yinterceptLabelDynamic">
           <property name="text">
            <string>Not calculated</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="calibrateButton">
       <property name="text">
        <string>Calibrate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
